### PR TITLE
Docs: update Sink: Mysql, s3 Source: mysql, s3, sns English

### PR DESCRIPTION
**Pull request by Michaelg22**

### CODE REVIEW

File: user-manual/connectors/sink/mysql.md [hunk 1]
```diff
@@ -1,7 +1,163 @@
...
+This quick start will guide you through the process of running an MySQL Sink connector.
+
+### Prerequisites
+- A running [MySQL][mysql] database.
+- Created db and tables.
+
+### Set MySql Sink Configurations
+You can specify your configs by either setting environments 
+variables or mounting a config.json to `/vance/config/config.json` 
+when running the connector.
```
Comment (VedaGao) [inline user-manual/connectors/sink/mysql.md:65]:
```suggestion
when running the Connector.
```

File: user-manual/connectors/sink/mysql.md [hunk 2]
```diff
@@ -1,7 +1,163 @@
...
+```
+ #### Config Fields of the MySql Sink
+| name             | requirement | description                                                                |
+|------------------|-------------|----------------------------------------------------------------------------|
+| v_port           | optional    | v_port is used to specify the port MySql Sink is listening on,default 8080 |
+| table_name       | required    | db table name                                                              |
+| insert_mode      | optional    | insert mode: insert or upsert, default insert                              |
+| commit_interval  | optional    | batch data commit to db interval, unit is millisecond default 1000         |
+
+### MySql Sink Secrets
```
Comment (VedaGao) [inline user-manual/connectors/sink/mysql.md:85]:
```suggestion
### MySQL Sink Secrets
```

File: user-manual/connectors/sink/mysql.md [hunk 3]
```diff
@@ -1,7 +1,163 @@
...
+
+Users should set their sensitive data Base64 encoded in a secret file.
+And mount your local secret file to `/vance/secret/secret.json` when you run the connector.
```
Comment (VedaGao) [inline user-manual/connectors/sink/mysql.md:88]:
```suggestion
And mount your local secret file to `/vance/secret/secret.json` when you run the Connector.
```

File: user-manual/connectors/sink/mysql.md [hunk 4]
```diff
@@ -1,7 +1,163 @@
...
+
+#### Encode your sensitive data
```
Comment (VedaGao) [inline user-manual/connectors/sink/mysql.md:90]:
```suggestion
#### Encode your Sensitive Data
```

File: user-manual/connectors/sink/mysql.md [hunk 5]
```diff
@@ -1,7 +1,163 @@
...
+Replace MY_SECRET with your sensitive data to get the Base64-based string.
+```shell
+$ echo -n MY_SECRET | base64
+QUJDREVGRw==
+```
+
+
+Here is an example of a Secret file for the MySQL Sink.
```
Comment (VedaGao) [inline user-manual/connectors/sink/mysql.md:98]:
```suggestion
Here is an example of a secret file for the MySQL Sink.
```

File: user-manual/connectors/sink/mysql.md [hunk 6]
```diff
@@ -1,7 +1,163 @@
...
+{
+  "host": "TVlfU0VDUkVUTVlfU0VDUkVU",
+  "port": "OTA4Mw==",
+  "username": "bG92ZWNob2NvbGF0ZQ==",
+  "password": "MTIzNDU2Nzg5",
+  "dbName": "SW1XYWxraW5PblN1blNoaW5l"
+}
+```
+
+#### Secret Fields of the Mysql Sink
```
Comment (VedaGao) [inline user-manual/connectors/sink/mysql.md:109]:
```suggestion
#### Secret Fields of the MySQL Sink
```

File: user-manual/connectors/sink/mysql.md [hunk 7]
```diff
@@ -1,7 +1,163 @@
...
+| dbName             | required    | db database name   |
+
+### Run the MySQL Sink with Docker
+Create your config.json and secret.json, and mount them to
+specific paths to run the MySQL Sink using the following command.
+
+> docker run -v $(pwd)/secret.json:/vance/secret/secret.json -v $(pwd)/config.json:/vance/config/config.json --rm vancehub/sink-mysql
+
+### Verify the MySQL Sink 
+You can verify if the MySQL Sink works properly by Sending a CloudEvent with the post command for example.
```
Comment (VedaGao) [inline user-manual/connectors/sink/mysql.md:126]:
```suggestion
You can verify if the MySQL Sink works properly by sending a CloudEvent with the post command for example.
```

**Review by VedaGao — CHANGES_REQUESTED** [review]

File: user-manual/connectors/sink/mysql.md [hunk 8]
```diff
@@ -1,7 +1,163 @@
...
+  "data" : {
+    "id":18,
+    "name":"xdl",
+    "description":"Development Manager",
+    "date": "2022-07-06"
+  }
+}"
+http://localhost:8081 
+```
+:::tip
```
Comment (VedaGao) [inline user-manual/connectors/sink/mysql.md:157]:
```suggestion
:::Tip

```

**Review by VedaGao — CHANGES_REQUESTED** [review]

File: user-manual/connectors/source/mysql.md [hunk 9]
```diff
@@ -1,7 +1,132 @@
 ---
 title: MySQL
-id: mysql
 ---
 
-# MySQL
-Coming soon
\ No newline at end of file
+# MySql Source
```
Comment (VedaGao) [inline user-manual/connectors/source/mysql.md:5]:
```suggestion
# MySQL Source
```

File: user-manual/connectors/source/mysql.md [hunk 10]
```diff
@@ -1,7 +1,132 @@
...
+This document provides a brief introduction of the MySQL Source.
+It is also designed to guide you through the process of running an
+MySQL Source Connector.
+## Introduction
+The MySql Source is a [Vance Connector][vc] that use [Debezium][debezium] 
+to obtain a snapshot of the existing data in a MySql database and then 
+monitors and record all subsequent row-level changes to that data.
```
Comment (VedaGao) [inline user-manual/connectors/source/mysql.md:12]:
```suggestion
monitors and records all subsequent row-level changes to that data.
```

File: user-manual/connectors/source/mysql.md [hunk 11]
```diff
@@ -1,7 +1,132 @@
...
+    "description": "Development Manager",
+    "date": "2022-07-06"
+  }
+}
+```
+## Quick Start
+This quick start will guide you through the process of running an MySQL Sink connector.
+
+### Prerequisites
+- A running [MySQL][mysql] database.
```
Comment (xdlbdy) [inline user-manual/connectors/source/mysql.md:59]: you need add link like "[mysql]:  https:xxxxx" at the document end

File: user-manual/connectors/source/mysql.md [hunk 12]
```diff
@@ -1,7 +1,132 @@
...
 
-# MySQL
-Coming soon
\ No newline at end of file
+# MySql Source
+This document provides a brief introduction of the MySQL Source.
+It is also designed to guide you through the process of running an
+MySQL Source Connector.
+## Introduction
+The MySql Source is a [Vance Connector][vc] that use [Debezium][debezium]
```
Comment (VedaGao) [inline user-manual/connectors/source/mysql.md:10]:
```suggestion
The MySQL Source is a [Vance Connector][vc] that use [Debezium][debezium] 
```

File: user-manual/connectors/source/mysql.md [hunk 13]
```diff
@@ -1,7 +1,132 @@
...
+# MySql Source
+This document provides a brief introduction of the MySQL Source.
+It is also designed to guide you through the process of running an
+MySQL Source Connector.
+## Introduction
+The MySql Source is a [Vance Connector][vc] that use [Debezium][debezium] 
+to obtain a snapshot of the existing data in a MySql database and then 
+monitors and record all subsequent row-level changes to that data.
+
+For example, MySql database vance has table vance_test Look:
```
Comment (VedaGao) [inline user-manual/connectors/source/mysql.md:14]:
```suggestion
For example, MySQL database Vance [https://docs.linkall.com/concepts/connector] has table vance_test Look:
```

File: user-manual/connectors/source/mysql.md [hunk 14]
```diff
@@ -1,7 +1,132 @@
...
+| port               | required    | db port                                                                                                    |
+| username           | required    | db username                                                                                                |
+| password           | required    | db password                                                                                                |
+| dbName             | required    | db database name                                                                                           |
+
+### Run the MySQL Sink with Docker
+Create your config.json and secret.json, and mount them to
+specific paths to run the MySQL Source using the following command.
+
+> docker run -v $(pwd)/secret.json:/vance/secret/secret.json -v $(pwd)/config.json:/vance/config/config.json --rm vancehub/source-mysql
```
Comment (xdlbdy) [inline user-manual/connectors/source/mysql.md:120]:
```shell
docker run -v $(pwd)/config.json:/vance/config/config.json -v $(pwd)/data:/vance/data --rm vancehub/source-mysql
```

File: user-manual/connectors/source/mysql.md [hunk 15]
```diff
@@ -1,7 +1,132 @@
...
+  "data": {
+    "id": 18,
+    "name": "xdl",
+    "description": "Development Manager",
+    "date": "2022-07-06"
+  }
+}
+```
+## Quick Start
+This quick start will guide you through the process of running an MySQL Sink connector.
```
Comment (VedaGao) [inline user-manual/connectors/source/mysql.md:56]:
```suggestion
This quick start will guide you through the process of running an MySQL Sink Connector.
```

File: user-manual/connectors/source/mysql.md [hunk 16]
```diff
@@ -1,7 +1,132 @@
...
+
+### Prerequisites
+- A running [MySQL][mysql] database.
+
+### Set MySql Source Configurations
```
Comment (VedaGao) [inline user-manual/connectors/source/mysql.md:61]:
```suggestion
### Set MySQL Source Configurations
```

File: user-manual/connectors/source/mysql.md (hunk 12 above)
Comment (xdlbdy) [inline user-manual/connectors/source/mysql.md:10]: all use MySQL, what do you think?

File: user-manual/connectors/sink/mysql.md [hunk 17]
```diff
@@ -1,7 +1,163 @@
...
 title: MySQL
-id: mysql
 ---
 
-# MySQL
-Coming soon
\ No newline at end of file
+# MySQL Sink
+This document provides a brief introduction of the MySQL Sink. 
+It is also designed to guide you through the process of running an
```
Comment (VedaGao) [inline user-manual/connectors/sink/mysql.md:7]:
```suggestion
It is also designed to guide you through the process of running a
```

File: user-manual/connectors/source/mysql.md [hunk 18]
```diff
@@ -1,7 +1,132 @@
...
++-------------+--------------+------+-----+---------+----------------+
+| Field       | Type         | Null | Key | Default | Extra          |
++-------------+--------------+------+-----+---------+----------------+
+| id          | int          | NO   | PRI | NULL    | auto_increment |
+| name        | varchar(100) | NO   |     | NULL    |                |
+| description | varchar(100) | NO   |     | NULL    |                |
+| date        | date         | YES  |     | NULL    |                |
++-------------+--------------+------+-----+---------+----------------+
+```
+
```
Comment (xdlbdy) [inline user-manual/connectors/source/mysql.md:26]: I think it needs to tell the user, the connector will transform MySQL record to a CloudEvent like below

File: user-manual/connectors/sink/mysql.md [hunk 19]
```diff
@@ -1,7 +1,163 @@
...
+
+## Introduction
+The MySql Sink is a [Vance Connector][vc] that aims to handle
+incoming CloudEvents in a way that extracts the data part of the 
+original event and delivers these extracted data to the MySQL database 
+by using JDBC. You will need to create a database and the tables before using 
+this Sink.
+
+## Handling incoming CloudEvent 
+For example, if the incoming CloudEvent looks like:
```
Comment (VedaGao) [inline user-manual/connectors/sink/mysql.md:18]:
```suggestion
For example, if the incoming CloudEvent looks like this:
```

File: user-manual/connectors/source/mysql.md [hunk 20]
```diff
@@ -1,7 +1,132 @@
...
+### Prerequisites
+- A running [MySQL][mysql] database.
+
+### Set MySql Source Configurations
+You can specify your configs by either setting environments
+variables or mounting a config.json to `/vance/config/config.json`
+when running the connector.
+
+Here is an example of a configuration file for the MySQL Source.
```
Comment (xdlbdy) [inline user-manual/connectors/source/mysql.md:66]: I think the example needs to be put below Config Fields

File: user-manual/connectors/sink/mysql.md [hunk 21]
```diff
@@ -1,7 +1,163 @@
...
+
+```json
+{
+  "v_port": "8081",
+  "table_name": "Costumer",
+  "insert_mode": "insert",
+  "commit_interval": "100" 
+}
+```
+ #### Config Fields of the MySql Sink
```
Comment (VedaGao) [inline user-manual/connectors/sink/mysql.md:77]:
```suggestion
 #### Config Fields of the MySQL Sink
```

**Review by VedaGao — CHANGES_REQUESTED** [review]

File: user-manual/connectors/sink/mysql.md (hunk 8 above)
Comment (Michaelg22) [reply user-manual/connectors/sink/mysql.md:157]: its a design tool

Comment (Michaelg22) [reply user-manual/connectors/sink/mysql.md:157]: please resolve this change

File: user-manual/connectors/source/mysql.md [hunk 22]
```diff
@@ -1,7 +1,132 @@
...
 title: MySQL
-id: mysql
 ---
 
-# MySQL
-Coming soon
\ No newline at end of file
+# MySql Source
+This document provides a brief introduction of the MySQL Source.
+It is also designed to guide you through the process of running an
```
Comment (VedaGao) [inline user-manual/connectors/source/mysql.md:7]:
```suggestion
It is also designed to guide you through the process of running a
```

File: user-manual/connectors/source/mysql.md [hunk 23]
```diff
@@ -1,7 +1,132 @@
 ---
 title: MySQL
-id: mysql
 ---
 
-# MySQL
-Coming soon
\ No newline at end of file
+# MySql Source
+This document provides a brief introduction of the MySQL Source.
```
Comment (VedaGao) [inline user-manual/connectors/source/mysql.md:6]:
```suggestion
This document provides a brief introduction of MySQL Source.
```

File: user-manual/connectors/source/mysql.md (hunk 20 above)
Comment (VedaGao) [inline user-manual/connectors/source/mysql.md:66]:
```suggestion
Here is an example of a configuration file for MySQL Source.
```

File: user-manual/connectors/source/mysql.md [hunk 24]
```diff
@@ -1,7 +1,132 @@
...
+| include_table      | optional    | comma-separated list of include table name, not include database name                                                                          |
+| exclude_table      | optional    | comma-separated list of exclude table name, not include database name. <br/>no need add system table and only no config include_table will use |
+| store_offset_key   | optional    | offset store use key, default is vance_debezium_offset                                                                                         |
+| offset_binlog_file | optional    | binlog filename, increment sync start binlog file name if not set is full sync                                                                 |
+| offset_binlog_pos  | optional    | binlog position, use with config offset_binlog_file                                                                                            |
+
+## MySql Source Secrets
+Users should set their sensitive data Base64 encoded in a secret file.
+And mount your local secret file to `/vance/secret/secret.json` when you run the connector.
```
Comment (VedaGao) [inline user-manual/connectors/source/mysql.md]:
```suggestion
And mount your local secret file to `/vance/secret/secret.json` when you run the Connector.
```

File: user-manual/connectors/source/mysql.md [hunk 25]
```diff
@@ -1,7 +1,132 @@
...
+
+### Encode your sensitive data
```
Comment (VedaGao) [inline user-manual/connectors/source/mysql.md]:
```suggestion
### Encode your Sensitive Data
```

File: user-manual/connectors/source/mysql.md [hunk 26]
```diff
@@ -1,7 +1,132 @@
...
+Replace MY_SECRET with your sensitive data to get the Base64-based string.
+```shell
+$ echo -n MY_SECRET | base64
+QUJDREVGRw==
+```
+
+Here is an example of a Secret file for the MySQL Sink.
```
Comment (VedaGao) [inline user-manual/connectors/source/mysql.md]:
```suggestion
Here is an example of a secret file for the MySQL Sink.
```

**Review by VedaGao — CHANGES_REQUESTED** [review]

**Review by VedaGao — APPROVED** [review]

**Pull request merged**